sendgrid/sendgrid.py

```diff
@@ -15,10 +14,8 @@
 
 import os
 import warnings
-
 import python_http_client
 
-
 class SendGridAPIClient(object):

```

Comment:
if you'll use a good linter like [python-black](https://github.com/psf/black). It'll automatically handle all these things and also the two-line gap before python classes are best practices. You may have a look at other files too.